parse_input_file.cc

```diff
@@ -155,17 +155,33 @@ namespace input_file_parsing {
 
         parse_geometry(nd, pparams);
 
-        // beta value from the gauge action
+
+        // beta, xi value from the gauge action
         Yp::read_verb<double>(pparams.beta, nd["begin_monomials"]["gauge"], "beta");
+        Yp::read_verb<double>(pparams.xi, nd["begin_monomials"]["gauge"], "xi");
```

Comment:
I'd change this as follows. 
- ```xi``` and ```anisotrope``` should be parsed with ```read_opt_verb()``` because are optional parameters.
- Since the ```xi``` parameter is needed only for anisotropic lattices, I'd parse it only if the ```anisotrope``` node exists.